package.json

```diff
@@ -8,6 +8,7 @@
   },
   "scripts": {
     "build": "make build",
```

Comment:
What is the purpose of this `prepare` script? It's identical to the `build` one.

Reply:
Hi @BrandonHBodine -- this change is optional, but it lets us reference unmerged branches when working on dependency upgrades [like this](https://github.com/openedx/frontend-app-course-authoring/pull/1052/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R48).

> Since npm v5.0.0 ([ref](https://blog.npmjs.org/post/161081169345/v500)):
> 
>> git dependencies with `prepare` scripts will have their `devDependencies` installed, and `npm install` run in their directory before being packed.